scripts/finetune.py

```diff
@@ -61,6 +77,8 @@ def get_multi_line_input() -> Optional[str]:
 
 
 def do_inference(cfg, model, tokenizer, prompter: Optional[str]):
+    if prompter == "None":
+        prompter = None
```

Comment:
Why is this needed?

Reply:
we have in the existing docs --prompter=None which I believe translates to a literal string. we could alternatively update the doc to be simply `--prompter` or alternatively default the value to None (which is what we're doing here)